app/test/vesting_test.go

```diff
@@ -0,0 +1,936 @@
+package app_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/app/encoding"
+	"github.com/celestiaorg/celestia-app/test/util/testfactory"
+	"github.com/celestiaorg/celestia-app/test/util/testnode"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	vestingtypes "github.com/cosmos/cosmos-sdk/x/auth/vesting/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	distributiontypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	tmtime "github.com/tendermint/tendermint/types/time"
+	"google.golang.org/grpc"
+)
+
+const (
+	totalAccountsPerType = 300
+	initBalanceForGasFee = 10
+)
+
+type _accountDispenser struct {
```

Comment:
Thanks for explaining! In Go any field that starts with a lowercase letter is unexported so I don't think the leading `_` is necessary or desirable.

Reply:
agree with rootul here, unless we're generating go code or have some extra reason, I think we should stick to standard go practices since that's what the rest of the repo history does :slightly_smiling_face: